vault/provider.go

```diff
@@ -400,6 +404,22 @@ var (
 			Resource:      identityGroupPoliciesResource(),
 			PathInventory: []string{"/identity/lookup/group"},
 		},
+		"vault_identity_oidc": {
+			Resource:      identityOidc(),
+			PathInventory: []string{"/identity/oidc/config"},
+		},
+		"vault_identity_oidc_key": {
+			Resource:      identityOidcKey(),
+			PathInventory: []string{"/identity/oidc/key/{name}"},
+		},
+		"vault_identity_oidc_key_allowed_client_id": {
+			Resource:      identityOidcKeyAllowedClientId(),
+			PathInventory: []string{"/identity/oidc/key/{key_name}"},
```

Comment:
Can you point me towards the API docs for this one?

Reply:
Please see [comment](https://github.com/terraform-providers/terraform-provider-vault/pull/488#issuecomment-518463146).